sdk/communication/AzureCommunicationCommon/Source/Identifiers.swift

```diff
@@ -75,17 +80,36 @@ import Foundation
      */
     public init(phoneNumber: String, rawId: String? = nil) {
         self.phoneNumber = phoneNumber
-        self.rawId = rawId
+        if let rawId = rawId {
+            self.rawId = rawId
+        } else {
+            let strippedPhoneNumber = phoneNumber.replacingOccurrences(of: "+", with: "")
+            self.rawId = "4:" + strippedPhoneNumber
+        }
```

Comment:
Previously we allowed developers to initialize without a `rawId` and it would just be optional. 
Now we are updating the logic, in case they do or formerly passed in `nil` we will still create a `rawId` on their behalf using the phone number that is passed in.